### PR TITLE
[gardening] Remove commented out code.

### PR DESCRIPTION
**Pull request by practicalswift**
#### What's in this pull request?

Remove commented out code.
#### Resolved bug number: –

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| All supported platforms | @swift-ci Please smoke test and merge |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

**Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| All supported platforms | @swift-ci Please test and merge |
| OS X platform | @swift-ci Please test OS X platform |
| OS X platform | @swift-ci Please benchmark |
| Linux platform | @swift-ci Please test Linux platform |

**Lint Testing**

| Language | Comment |
| --- | --- |
| Python | @swift-ci Please Python lint |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

File: lib/IDE/Formatting.cpp [hunk 1]
```diff
@@ -550,12 +550,6 @@ class FormatWalker : public SourceEntityWalker {
         // FIXME: We are going to revisit the behavior and the indentation we
```
Comment (gribozavr) [inline lib/IDE/Formatting.cpp]: I'm not sure if the FIXME is related to the commented out code.

**Comment by gribozavr** [comment]
CC @benlangmuir @akyrtzi @nkcsgexi

**Comment by benlangmuir** [comment]
This code was commented out in d10220920b278f0ba3de31691f2e8dd9bea44462 by Argyrios at the same time he added the comment and FIXME.  I'm assigning to @akyrtzi.

**Comment by akyrtzi** [comment]
FIXME is related, I'd prefer to keep the commented-out part.

**Comment by practicalswift** [comment]
@akyrtzi PR updated. Now keeping the code and switching to the preferred form `#ifdef 0` instead of `/*` :-)

**Comment by akyrtzi** [comment]
LGTM, thanks!

**Comment by akyrtzi** [comment]
Should have been "#if 0" not "#ifdef 0". Fixed it in subsequent commit 49d786557c06c6d0b861315490ef7e45ae9b474d

**Comment by practicalswift** [comment]
@akyrtzi Thanks for fixing and sorry for my thinko!

**Pull request merged**